src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1062,6 +1070,14 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='Generate and assign an Azure Active Directory Identity for this server'
                    'for use with key management services like Azure KeyVault.')
 
+        c.argument('public_network_access',
+                   options_list=['--public-network-access', '-a'],
```

Comment:
What about using something like `--enable-public-network` for the argument?
I have two questions here:
1. What is the default network access policy for sql server? public or private?
2. If using private link, should users create private endpoint connection by themselves?

Reply:
Do you mean a flag?  So --enable-public-network means Enabled while no flag means disabled?  If so, two things about this, first off powershell uses PublicNetworkAcces and takes in a string Enabled/Disabled.  I wanted to keep it consistent between the two.  Also, we would want a third option, None,  where no change to PublicNetworkAccess occurs and if we use a flag, we wouldn't be able to have that.

In response to the other questions:
1. Default is Public (aka PublicNetworkAccess == Enabled)
2. Yes, user has to create private endpoint connection by themselves